docroot/modules/humsci/hs_layouts/hs_layouts.layouts.yml

```diff
@@ -24,6 +24,8 @@ three_column_w_image:
   icon_map:
     - [sidebar_first, main, sidebar_second]
   regions:
+    title:
```

Comment:
This should have a respective change to the template in the layout module as well.

Reply:
@pookmish This should be good to go now.

Reply:
Thank you